presto-main/src/main/java/io/prestosql/spiller/FileSingleStreamSpillerFactory.java

```diff
@@ -78,7 +78,9 @@
     private final double maxUsedSpaceThreshold;
     private final boolean spillEncryptionEnabled;
     private int roundRobinIndex;
-    private final LoadingCache<Path, Boolean> spillPathHealthCache;
+
+    @VisibleForTesting
+    final LoadingCache<Path, Boolean> spillPathHealthCache;
```

Comment:
Maybe instead of exposing a field, create a method that exposes just the size of cache.

Reply:
It looks like this comment is not addressed

Reply:
Ah, didn't push the last commit. Fixed.